lib/otel/segments/consumer.js

```diff
@@ -40,6 +41,7 @@ function createConsumerSegment(agent, otelSpan) {
   transaction.name = segmentName
```

Comment:
this needs to be `transaction.setPartialName(segmentName)`. it's responsible for naming the `'OtherTransactionTotalTime/Message` metric name